src/vt-tv/api/info.h

```diff
@@ -2,10 +2,10 @@
 //@HEADER
 // *****************************************************************************
 //
-//                                  info.h
+//                                    info.h
 //             DARMA/vt-tv => Virtual Transport -- Task Visualizer
 //
-// Copyright 2019 National Technology & Engineering Solutions of Sandia, LLC
+// Copyright 2019-2024 National Technology & Engineering Solutions of Sandia, LLC
```

Comment:
@nlslatt  The `license-template` in vt has the copyright from `2019-2021`--I changed this to 2019-2024. Is this correct, or should I stay with 2019-2021?

Reply:
Sounds reasonable, but I don't know what the official answer is

Reply:
We discussed this in the meeting and decided to use `2019-2024`